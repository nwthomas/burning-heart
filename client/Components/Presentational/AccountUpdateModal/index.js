export { default as AccountUpdateModal } from "./AccountUpdateModal";
