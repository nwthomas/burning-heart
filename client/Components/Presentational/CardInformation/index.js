export { default as CardInformation } from "./CardInformation";
