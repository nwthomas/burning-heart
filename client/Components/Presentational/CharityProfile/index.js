export { default as CharityProfile } from "./CharityProfile";
