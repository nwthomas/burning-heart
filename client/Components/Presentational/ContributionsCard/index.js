export { default as ContributionsCard } from "./ContributionsCard";
