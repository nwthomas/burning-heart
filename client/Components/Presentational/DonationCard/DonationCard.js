import React from "react";
import { Text, View, StyleSheet } from "react-native";

const DonationCard = ({ donation }) => {
  return (
    <View style={styles.cardContainer}>
      <Text style={styles.donationCardText}>{donation.charityName}</Text>
      <Text style={styles.donationAmount}>{`$${parseFloat(
        donation.amount / 100
      ).toFixed(2)}`}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  cardContainer: {
    alignSelf: "stretch",
    paddingTop: 30,
    paddingBottom: 30,
    borderBottomColor: "#cacacf",
    borderBottomWidth: 3
  },
  donationCardText: {
    fontFamily: "Montserrat-Medium",
    fontSize: 16
  },
  donationAmount: {
    fontFamily: "OpenSans-Italic",
    fontSize: 14,
    marginTop: 5
  }
});

export default DonationCard;
