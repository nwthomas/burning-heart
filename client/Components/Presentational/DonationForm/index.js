export { default as DonationForm } from "./DonationForm";
