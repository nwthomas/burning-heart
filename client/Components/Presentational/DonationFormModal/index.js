export { default as DonationFormModal } from "./DonationFormModal";
