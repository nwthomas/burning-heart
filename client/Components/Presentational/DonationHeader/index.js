export { default as DonationHeader } from "./DonationHeader";
