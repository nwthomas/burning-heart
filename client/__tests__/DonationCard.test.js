import React from "react";
import "react-native";
import { DonationCard } from "../Components/Presentational/DonationCard";
import renderer from "react-test-renderer";

describe("<DonationCard />", () => {
  it("should render the component without crashing", () => {
    const donation = { charityName: "name", amount: 10 };
    const helpers = renderer.create(<DonationCard donation={donation} />);
  });

  it("should match the snapshot of the tree", () => {
    const donation = { charityName: "name", amount: 10 };
    const tree = renderer.create(<DonationCard donation={donation} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
