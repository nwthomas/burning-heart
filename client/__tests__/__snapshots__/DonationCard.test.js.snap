// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DonationCard /> should match the snapshot of the tree 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "borderBottomColor": "#cacacf",
      "borderBottomWidth": 1,
      "paddingBottom": 30,
      "paddingTop": 30,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Roboto-Medium",
        "fontSize": 16,
      }
    }
  >
    name
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "Roboto-Medium",
        "fontSize": 14,
        "marginTop": 5,
      }
    }
  >
    $0.10
  </Text>
</View>
`;
